contributors/design-proposals/api-machinery/admission-control-webhooks.md

```diff
@@ -256,7 +259,7 @@ We encourage the use of patch to avoid the "old clients dropping new fields" pro
 
 ## Bootstrapping 
 
-We will add a WebhookConfig.Webhooks.exemptNamespace (or  an WebhookConfig.Webhooks.Rule.exemptNamespace?  This can exempt  some namespaces from webhooks (See #51290) for bootstrapping.
+We will add a WebhookConfig.Webhooks.exemptNamespace (or  an WebhookConfig.Webhooks.Rule.exemptNamespace?  This can exempt some namespaces from webhooks (See #51290) for bootstrapping.
```

Comment:
Chao has a pull somewhere (one these tabs) describing his proposal.  Find that one and link it?

Reply:
done